.github/workflows/add-triage-label.yml

```diff
@@ -13,10 +17,41 @@ jobs:
       - name: Run
         uses: actions/github-script@v7
         with:
-          script: |
-            github.rest.issues.addLabels({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              labels: ["status: needs triage"]
-            })
\ No newline at end of file
+          script: | 
+            const INTERNAL_LABELS = ["Internal"];
```

Comment:
Please add here `status: triaged` as well.

(it really should be `status: triaged` only, but since I can't convince folks to follow the process, we'll probably have to rework it, so let's keep `Internal` as well)